### PR TITLE
Jw/usesdk

### PR DESCRIPTION
**Pull request by jarrodwatts**

### CODE REVIEW

File: src/Provider.tsx [hunk 1]
```diff
@@ -508,7 +508,12 @@ export const ThirdwebSDKProvider: React.FC<
 /**
  *
  * @returns {@link ThirdwebSDK}
- * @internal
+ * Create an instance of the Thirdweb SDK that uses the connected wallet to sign transactions,
```
Comment (jnsdls) [inline src/Provider.tsx:511]: this does not *create* an instance of the SDK but rather it accesses the SDK instance created by `<ThirdwebProvider>`

**Review by jnsdls — CHANGES_REQUESTED** [review]
make it clear useSDK accesses the sdk not creates

File: src/Provider.tsx (hunk 1 above)
Comment (jarrodwatts) [reply src/Provider.tsx:511]:
```suggestion
 * Access the instance of the thirdweb SDK created by the ThirdwebProvider, that uses the connected wallet to sign transactions,
```

File: src/Provider.tsx [hunk 2]
```diff
@@ -508,7 +508,12 @@ export const ThirdwebSDKProvider: React.FC<
...
+ * and the desiredChainId of the ThirdwebProvider context to determine the network to use.
```
Comment (jarrodwatts) [inline src/Provider.tsx:512]:
```suggestion
 * on the defined desiredChainId.
```
```suggestion
 * on the desiredChainId
```

**Comment by jarrodwatts** [comment]
updated @jnsdls

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**